### DISCUSSION TITLE
Floorp =< 11.16 get disabled on Mojave - why? Is there a workaround?

### DISCUSSION THREAD

**Discussion by dajare** [Floorp-Projects/Floorp#1507] [Q&A] [answered]
As noted in a [bug comment](https://github.com/Floorp-Projects/Floorp/issues/1348#issuecomment-2295082817), from v. 11.17, Floorp is based on ESR 128 so that Mojave (Macos 10.14.6) is no longer supported.

Floorp versions =< 11.16, then, *should* run on Mojave. They are based on ESR 115, and I have confirmed that ESR 115 runs perfectly fine on Mojave.

The issue with Floorp =< 11.16 (I have tried 11.16, 11.15, 11.14.1, and 11.14) is that they _install_ just fine. It runs, you browse and enjoy Floorp goodness. **But!** after closing the app *it will not reopen*, and instead one sees this:

<img width="437" alt="floorp-no" src="https://github.com/user-attachments/assets/ccfe1840-a535-4cbe-9d80-4b00d980ae80">

This is very puzzling! Since FF ESR 115 runs perfectly fine under Mojave, it *seems* that Floorp is making some checks on **re-opening** that are not made at installation, those checks are failed, and Floorp does not run.

This is odd! I understand that there is no responsibility to maintain "old" versions, but perhaps someone knows a simple fix for this, or a workaround, that will permit continued use of Floorp =< 11.16 on Mojave. If not, well, then there is at least FF ESR 115 itself.

Thanks for any help with this!

**Comment by typeling1578** [comment, marked as answer]
Perhaps Floorp is being automatically updated.

Possible workaround steps.
1. Install an older version of Floorp
2. Disconnect from the Internet.
3. Start Floorp
4. Open `about:config` and set `app.update.auto` to false
5. Restart Floorp
6. Connect to the Internet

**Reply by dajare** [reply]
Nailed it! :) Thank you. A follow-up? So Floorp 11.16 gives options to auto-install, OR check-and-ask:

<img width="432" alt="floorp-chk" src="https://github.com/user-attachments/assets/69cf3ba1-0841-4d61-b322-bb6563985d2f">

Since I'm "stuck" on 11.16, is there an advanced option (`about:config`) that will prevent me from having get the pop-up "Update available" which I then need to "Dismiss" on every re-launch? I tried toggling a couple possibilities (`browser.search.update`, `enable.floorp.update`), but those didn't prevent the pop-up invitation. :/

Thanks again for the help!

**Reply by typeling1578** [reply]
The easiest way to completely disable the update feature would be to use Enterprise Policies.

These documents are helpful:
https://github.com/mozilla/policy-templates/tree/master/mac
https://mozilla.github.io/policy-templates/#disableappupdate

**Reply by dajare** [reply]
Thanks @typeling1578 - that helped enormously! I ran these two command line instructions to have this on a "user" basis (no `sudo` required):

```
defaults write ~/Library/Preferences/org.mozilla.firefox EnterprisePoliciesEnabled -bool TRUE
defaults write ~/Library/Preferences/org.mozilla.firefox DisableAppUpdate -bool TRUE
```

And that did the trick. :) Thanks again for your help! Maybe this will help others, too (there are a lot of hits for "best browser mojave" in a web search)!
